comid-triples.cddl

```diff
@@ -29,24 +31,43 @@ endorsed-triple-record = [
   one-or-more<measurement-map> ; endorsed measurements
 ]
 
+; ATTESTATION-VERIFICATION-KEY triple
+; "attesting environment $AE" "signs Evidence that can be verified with key" "$K"
+attest-key-triple-record = [
+  environment-map ; attesting environment
+  one-or-more<verification-key> ; attestation verification key(s)
+]
+
 ; DEVICE-IDENTITY triple
-; (Should it be AE-IDENTITY instead?)
-; "attesting environment $AE" "signs Evidence with key" "$K"
+; "device $D" "is identified by key" "$K"
 identity-triple-record = [
-  environment-map ; attesting-environment
-  one-or-more<COSE_Key> ; identity
+  environment-map ; device identifier (instance or class)
+  one-or-more<verification-key> ; DevID, or semantically equivalent
 ]
 
+verification-key = {
+  ; Verification key (typically a public key)
+  corim.key => COSE_Key
+  ; Optional X.509 certificate chain corresponding to the public key
+  ; in corim.key, encoded as an array of one or more base64-encoded
+  ; DER PKIX certificates.  The certificate containing the public key
+  ; in corim.key MUST be the first certificate.  This MAY be followed
+  ; by additional ; certificates, with each subsequent certificate
+  ; being the one used to certify the previous one.
+  ? corim.keychain => [ + tstr ]
+}
+
 environment-map = non-empty<{
   ? comid.class => class-map
-  ? comid.instance => instance-map
+  ? comid.instance => $instance-id-type-choice
 }>
 
 class-map = {
   comid.class-id => $class-id-type-choice
   ? comid.layer => uint
   ? comid.index => uint
-  * $$class-map-extension
+  ? comid.vendor => tstr
```

Comment:
Just a minor comment, I would prefer to have the vendor and model first and then layer and index.